internal/gateway/connector/hooks/copilot-hook.sh

```
@@ -7,11 +7,16 @@ set -euo pipefail
 HOME="${HOME:-${USERPROFILE:-$(cd ~ 2>/dev/null && pwd)}}"
 export HOME
 
+HOOK_DIR="$(cd "$(dirname "${BASH_SOURCE[0]:-$0}")" && pwd)"
+{{if .Managed}}
```

Shellcheck (0.11.0)

[error] You need a space after the '{'.

(SC1054)